src/main/java/org/springframework/data/aerospike/cache/AerospikeCacheManager.java

```diff
@@ -63,9 +63,10 @@
 	 * set name "aerospike".
 	 * 
 	 * @param aerospikeClient the {@link AerospikeClient} instance.
+	 * @param aerospikeConverter
 	 */
-	public AerospikeCacheManager(AerospikeClient aerospikeClient) {
-		this(aerospikeClient, Collections.<String>emptyList());
+	public AerospikeCacheManager(AerospikeClient aerospikeClient, MappingAerospikeConverter aerospikeConverter) {
```

Comment:
I prefer to provide dependencies to the classes, not to hardcode them. Moreover library is named spring-data-*, so I suppose, it is obvious that it is going to be used inside ioc container. Do you agree?